SimpleMathJaxHooks.php

```diff
@@ -31,13 +31,11 @@ public static function renderChem($tex, array $args, Parser $parser, PPFrame $fr
 	}
 
 	private static function renderTex($tex, $parser) {
-		$parser->getOutput()->addModules( 'ext.SimpleMathJax' ); 
+		$parser->getOutput()->addModules( 'ext.SimpleMathJax' );
 		$parser->getOutput()->addModules( 'ext.SimpleMathJax.mobile' ); // For MobileFrontend
-		return ["<span style='opacity:.5'>[math]${tex}[/math]</span>", 'markerType'=>'nowiki'];
+		$attributes = [ "style" => "opacity:.5" ];
+		Hooks::run( "SimpleMathJaxAttr", [ &$attributes, $tex ] );
```

Comment:
Thanks for your feedback! Let's just add the hook 'SimpleMathJaxAttributes'.

```suggestion
		Hooks::run( "SimpleMathJaxAttributes", [ &$attributes, $tex ] );
```